org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/capabilities/XMLCapabilityManager.java

```diff
@@ -181,6 +184,11 @@ private void registerWatchedFiles() {
 		registerCapability(WORKSPACE_WATCHED_FILES_ID, WORKSPACE_WATCHED_FILES, options);
 	}
 
+	public void registerExecuteCommand(List<String> commands) {
+		registerCapability(WORKSPACE_EXECUTE_COMMAND_ID, WORKSPACE_EXECUTE_COMMAND,
```

Comment:
We don't seem to call `unregisterCapability` anywhere... if commands are registered again does this "registration over registration" works ok in VScode and Eclipse?